assembl/static2/js/app/pages/administration.jsx

```diff
@@ -265,6 +295,14 @@ export default compose(
       };
     }
   }),
+  graphql(SectionsQuery, {
+    props: ({ data }) => {
```

Comment:
you have to handle loading and errors here

Reply:
yes